packages/fx-core/src/common/spec-parser/adaptiveCardWrapper.ts

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+"use strict";
+
+import { AdaptiveCard, WrappedAdaptiveCard } from "./interfaces";
+
+export function wrapAdaptiveCard(
+  card: AdaptiveCard,
+  title: string,
+  subtitle: string
+): WrappedAdaptiveCard {
+  const result: WrappedAdaptiveCard = {
+    version: "devPreview",
+    $schema: "<URL_REFERENCE_TO_SCHEMA>",
```

Comment:
would be better if we could save the constants here in the "constants.ts"? thought the value will be changed later.

Reply:
Sure